### PR TITLE
Add pubu Action

### PR DESCRIPTION
**Pull request by trigged**
It can send message to PubuIM 
seems like 

image

### CODE REVIEW

File: lib/fastlane/actions/pubu.rb [hunk 1]
```diff
@@ -0,0 +1,80 @@
+module Fastlane
+  module Actions
+    # To share this integration with the other fastlane users:
+    # - Fork https://github.com/KrauseFx/fastlane
```
Comment (KrauseFx) [inline lib/fastlane/actions/pubu.rb]: Please remove those comments

File: lib/fastlane/actions/pubu.rb [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      end
+
+      def self.details
+        # Optional:
+        # this is your change to provide a more detailed description of this action
+        "You can use Pubu to Notification"
+      end
+
+      def self.authors
+        # So no one will ever forget your contribution to fastlane :) You are awesome btw!
```
Comment (KrauseFx) [inline lib/fastlane/actions/pubu.rb]: Please remove those comments

**Comment by ohwutup** [comment]
hey @trigged! thanks for submitting this pull request! 

if you don't mind, could you just take care of the comments pointed out by @KrauseFx and then the few outstanding hound issues and then we should be good to go! 

:rocket:

**Comment by trigged** [comment]
@ohwutup  @KrauseFx  thanks for mention , i will fix this issues as soon as possible

**Comment by ohwutup** [comment]
hi @trigged - it looks like there is just one more issue from hound https://github.com/fastlane/fastlane/pull/994#discussion-diff-47643717R31

File: lib/fastlane/actions/pubu.rb [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+        # Below a few examples
+        [
+          FastlaneCore::ConfigItem.new(key: :message,
+                                       description: "The message will displayed on PubuIM",
+                                       optional: true),
+
+          FastlaneCore::ConfigItem.new(key: :pubuUrl,
+                                       env_name: "PUBU_URL",
+                                       description: "PubuIM services url", #todo add doc url here
```
Comment (KrauseFx) [inline lib/fastlane/actions/pubu.rb]: In this case, the TODO should be even be removed by adding the URL /cc @ohwutup

**Comment by trigged** [comment]
@ohwutup  all done, should i rebase all commits in the one ?

**Comment by ohwutup** [comment]
@trigged that would be great, would you also mind adding in a simple test or two just to make sure that we are keeping our coverage up?

**Comment by ohwutup** [comment]
also, could you rebase from master as well to get the newest changes? that should fix the build in CI as well. thanks!

**Comment by trigged** [comment]
@ohwutup ok, i will change this as soon as possible

File: docs/Actions.md [hunk 4]
```diff
@@ -1667,6 +1667,27 @@ Post a message to a **group chat**.
...
+pubu(
+  message: "Ping!"
+)
+
+pubu(
+  message: "Ping!",
+  payload: {            # Optional. You can specify your own message attachments.
+    'Built by' => "Pubu"
+  },
+  color : "info"        # Optional. The value can either be one of info, warning, primary, error, muted or success. The default value is info."
```
Comment (ohwutup) [inline docs/Actions.md]: should there be a space here after `color`?

**Comment by ohwutup** [comment]
hey @trigged, one last little comment i missed, and then do you mind rebasing on master one more time and then squashing the commits? then i'll merge right away! :icecream:

File: docs/Actions.md [hunk 5]
```diff
@@ -1667,6 +1667,27 @@ Post a message to a **group chat**.
...
 
+### [PubuIM](http://pubu.im)
+Get a Webhook URL by adding a Fastlane service in your Pubu dashboard. Specify PUBU_URL to the URL, and then Fastlane can send notifications with results.
+
+
+```ruby
+ENV["PUBU_URL"] = "https://hooks.pubu.im/services/xxxx" # Define a WebHook URL
+
+pubu(
+  message: "Ping!"
```
Comment (KrauseFx) [inline docs/Actions.md]: Please change this to be one line only

Comment (KrauseFx) [inline docs/Actions.md]: pubu(message: "Ping!")

**Comment by ohwutup** [comment]
hey @trigged, do you mind just making that one last change @KrauseFx mentioned and squash the commits and we will be good to go!

**Comment by ohwutup** [comment]
I opened a new PR https://github.com/fastlane/fastlane/pull/1425 with the cleanup and commits squashed. Lets continue discussion over there if necessary!

**Pull request closed without merging**